packages/sanity/src/core/schema/createSchema.ts

```diff
@@ -6,6 +6,12 @@ import {inferFromSchema as inferValidation} from '../validation'
 
 const isError = (problem: SchemaValidationResult) => problem.severity === 'error'
 
+const builtinSchema = SchemaBuilder.compile({
+  name: 'studio',
```

Comment:
why changing this to `studio`?

Reply:
Changing? This is the registry for the Studio builtin types. This name isn't observable from the Schema returned from `createSchema`.